dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
```

Comment:
0.1.1
We should really just grunt replace this number in anywhere it needs to go